src/hooks/useModal/README.md

```diff
@@ -1,24 +1,27 @@
 ## Usage
 
-```
-import React, { useState, useCallback } from 'react';
-import { render } from 'react-dom';
-import useModal, { ModalRoot } from 'src/hooks/useModal';
-const App = () => {
-  const [Modal, open, close] = useModal('className');
+```tsx
+import React, { useCallback } from 'react';
+import { GlobalModalProvider, useGlobalModalContext } from 'src/hooks/useModal';
+
+export const CustomComponent = () => {
+  const { openModal } = useGlobalModalContext();
+
+  const showUpAlert = useCallback(() => {
+    openModal({
+      modalProps: {},
+      childElement: (
+        <div className="custom-alert">
+          "Button has been clicked!"
+        </div>
+      ),
+    });
+  }, []);
+
   return (
-    <div>
+    <div className="custom-button">
       <button onClick={open}>OPEN</button>
```

Comment:
where is `open`?

Reply:
This is my mistake :D I replace it with the `showUpAlert`